.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         # - xslt is needed for older Nokogiris, RUBY_VERSION < 2.5
         # - sasl is needed for memcached
       - name: Install OS packages
-        run: sudo apt-get install -y --no-install-recommends libcurl4-nss-dev libsasl2-dev libxslt1-dev
+        run: sudo apt-get update; sudo apt-get install -y --no-install-recommends libcurl4-nss-dev libsasl2-dev libxslt1-dev
```

Comment:
I merged a CI fix branch #1380 before it was merged into Dev so that my branch would pass.